view/base/web/js/sentry/sentry-config.js

```diff
@@ -29,7 +29,17 @@ define(
                         {
                             dsn: config.dsn,
                             beforeSend(event) {
-                                var isBreadIssue = event.extra && Object.values(event.extra).includes('BreadIssue');
+                                if (!event.extra) {
+                                    return null;
+                                }
+
+                                var isBreadIssue = false;
+
+                                Object.keys(event.extra).map(function(key) {
+                                    if (event.extra[key] === 'BreadIssue') {
+                                        isBreadIssue = true;
```

Comment:
Not necessary, but you could add a `return` here to prevent extra looping

Reply:
oh yeah great catch thanks!